### ISSUE TITLE
Plugin install lock file is not deleted

### ISSUE THREAD

**Issue by valleema** [nextflow-io/nextflow#2103]
### Expected behavior and actual behavior

When launched `-with-tower`, execution stops immediately with :
```
/tmp/nextflow-plugin-nf-tower-1.1.0.lock (Permission denied)
```
☝️ which is normal, as this file is owned by an other user.

### Steps to reproduce the problem

```
$ nextflow run hello -r e6d9427e5b -with-tower
N E X T F L O W  ~  version 21.04.0
Launching `nextflow-io/hello` [spontaneous_hugle] - revision: e6d9427e5b89b6e3c7641fe7528da1ae339d2334
/tmp/nextflow-plugin-nf-tower-1.1.0.lock (Permission denied)
```

### Program output 

[nextflow.log](https://github.com/nextflow-io/nextflow/files/6456865/nextflow.log)

### Environment 

* Nextflow version: 21.04.0
* Java version: openjdk version "1.8.0_212"
* Operating system: Debian GNU/Linux 9 (stretch)
* Bash version: GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)

**Comment by pditommaso** [comment]
ok, found the problem. thanks for reporting

**Comment by pditommaso** [comment]
Solved b7fdffac
